nss-cohort-36/keahua-arboretum-mythical-techies
# Mb feed animal

# Description

Feed animal functionality (#3 on the main menu)

Fixes #22 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout mb-feed-animal
Run index.py. 
Go through the feed animal functionality (option 3). 
See if it works appropriately for a couple of animals.

# Checklist:
